cmake/modules/RootBuildOptions.cmake

```diff
@@ -435,6 +435,12 @@ foreach(opt cxxmodules gfal gsl_shared jemalloc monalisa pyroot_legacy tcmalloc
   endif()
 endforeach()
 
+foreach(opt minuit2_omp minuit2_mpi)
+  if(${opt})
+      message(WARNING "The option '${opt}' is dangerous because it means you can only minimize thread-safe functions with Minuit2. This breaks other parts of ROOT, e.g. RooFit. If you want Minuit2 with OpenMP or MPI support, it is better to build Minuit2 standalone.")
```

Comment:
I would write: 

The option ... can only be used to minimise thread-safe functions in Minuit2. It cannot be used for Histogram/Graph fitting and for RooFit. If you want to use Minuit2 with OpenMP or MPI support, it is better to build Minuit2 as a standalone library.